lib/supply/uploader.rb

```diff
@@ -74,6 +74,14 @@ def upload_binary
         else
           client.upload_apk_to_track(Supply.config[:apk], Supply.config[:track])
         end
+
+        ## Obbs upload
+        search = File.join(File.dirname(Supply.config[:apk]), '*.obb')
+        paths = Dir.glob(search, FILE::FNM_CASEFOLD)
+        paths.sort.each do |path|
+          Helper.log.info "Uploading obb file #{path}..."
+          client.upload_obb(obb_file_path: File.expand_path(path))
```

Comment:
it s funny that we do an expand here, we don't do it for apk. WOnder if we re missing one for apk as well.